lib/tasks/scheduler.rake

```diff
@@ -26,5 +26,20 @@ end
 
 desc "Send monthly summary to Slack"
 task monthly_price_summary: :environment do
-  HappyHood::Slack::Client.send_monthly_price_summary
+  cache_key = "monthly_summary/#{Date.today.beginning_of_month}"
+  sent_message = false
+
+  Rails.cache.fetch(cache_key, expires_in: 1.month) do
```

Comment:
Because `fetch` returns the result if the cache key is set, `fetch` will return `true`. However, we only set `sent_message = true` the first time this runs. The next time, `sent_message` will remain false.

Reply:
I feel like this is all bit over engineered. Why wouldn't we just adjust the scheduler to run on the first of the month? Then we have no code change needed

Reply:
Heroku Scheduler does not support a cron-tab like expression. It only supports every 10 minute/hour/day interval: 
![image](https://user-images.githubusercontent.com/1649281/123561518-6a04b580-d76e-11eb-9998-3e28c3a2b264.png)

Reply:
Oh well there's my wrong assumption! Got it